packages/sui-lint/eslintrc.js

```diff
@@ -41,7 +41,68 @@ const TESTING_RULES = {
     RULES.ERROR,
     {allowShortCircuit: true, allowTernary: true}
   ],
-  'no-only-tests/no-only-tests': RULES.ERROR
+  'no-only-tests/no-only-tests': RULES.ERROR,
```

Comment:
This set of rules are taken from kentcdodds jest eslint config https://github.com/kentcdodds/eslint-config-kentcdodds/blob/main/jest.js#L48